b2sdk/api.py

```diff
@@ -63,6 +63,7 @@ def __init__(
         max_download_workers: Optional[int] = None,
         save_to_buffer_size: Optional[int] = None,
         check_download_hash: bool = True,
+        max_streams: Optional[int] = None,
```

Comment:
`max_download_streams_per_file or `max_streams_per_downloaded_file` - `max_streams` can mean a lot of different things and can be confusing